scripts/dev/uaa_knowledge.py

```
@@ -0,0 +1,315 @@
+    )
+    context.add_argument("--category")
+    context.add_argument("--collection")
+    context.add_argument("--tag")
+    return parser
+
+
+def main(argv: Sequence[str] | None = None) -> int:
+    args = build_parser().parse_args(argv)
+    try:
+        return _run(args)
+    except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)